corehq/apps/README.rst

```diff
@@ -157,7 +157,7 @@ These apps are developer-facing tools.
 
 cachehq
    Caching functinality for CouchDB.
-case_migrations
+case_migrations<Deprecated>
```

Comment:
You can delete this altogether - the code referenced here no longer exists. The notes above on the `ota` app already reference the restore, although you could update them to note that the ota app is used by both CommCare mobile and formplayer.

Reply:
Thanks Jenny! Fixed here: 170650dd76553f80934bd6ed344d91926a24489c